index.html

```diff
@@ -60,13 +60,164 @@ <h3>I can help you build a product , feature or website Look through some of my
+
+          <form action="#" class="project2" id="project21">
+            <h1 class="s4h1"> Profesional Art Printing Data</h1> 
+            <p class="p2">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+  
+                   
+
+          <ul class="lan2" >
+              <li><img src="image/html2.png" alt=""></li>
+              <li><img src="image/bootstrap2.png" alt=""></li>
+              <li><img src="image/ruby2.png" alt=""></li>
+
+          </ul>
+          <button class="btn2">See Project </button> </form>
+
+
+          <form action="#" class="project2" id="project20">
+            <h1 class="s4h1"> Profesional Art Printing Data</h1> 
+            <p class="p2">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+  
+                   
+
+          <ul class="lan2" >
+              <li><img src="image/html2.png" alt=""></li>
+              <li><img src="image/bootstrap2.png" alt=""></li>
+              <li><img src="image/ruby2.png" alt=""></li>
+
+          </ul>
 
+          <button class="btn2">See Project </button> </form>
+        
+                     
+    </div>
+           
+
+
+
+       <div class="body3">
+        <h1 class="s1h11">About me</h1>
+
+        <h3>Hello I’m a software developer! I can help you build a product 
+            , feature or website Look through some of my work and experience! If you like what you see and have
+             a project you need coded, don’t hestiate to contact me.</h3>
+    
+
+             <form class="form2" action="https://www.linkedin.com/in/abdelruhman-mihamed-a42667179/overlay/1635484731651/single-media-viewer/"><button class="btn11">Get My Resume </button> </form> 
+              
+             <hr class="hr2">
+             <div class="description">
+                 
+                
+                   <img src="image/lang.png" alt="">  
+                <ul class="ul44">
+                    <li class="lastli">Javascript</li>
+                    <li class="lastli">Ruby</li>
+                    <li class="lastli">Html</li>
+                    <li class="lastli">css</li>
+                </ul>  
+                
+                <img class="h4" src="image/frame.png" alt="">
+                <ul>
+                    <li class="lastli">Bootstrap</li>
+                    <li class="lastli">CapyBara</li>
+                    <li class="lastli">Selenium</li>
+                    <li class="lastli">RSPec</li>
+                </ul>
+
+                <img class="h4" src="image/skills.png" alt="">
+                <ul>
+                    <li class="lastli">Codepen</li>
+                    <li class="lastli">Terminal</li>
+                    <li class="lastli">codekit</li>
+                    <li class="lastli">Github</li>
+                    <li class="lastli">Gitlab</li>
+                </ul>
```

Comment:
- [ ] Please, try to align the languages, skills, and languages lists to the center to improve your UI design.
- Your design:
<img width="156" alt="image" src="https://user-images.githubusercontent.com/77190203/164861370-cc04e888-560f-48e2-a2b1-b2a70f8eba46.png">